store/test/store/products_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule Store.ProductsTest do
+  use Store.DataCase
+
+  alias Store.Products
+
+  describe "products" do
+    alias Store.Products.Product
+
+    import Store.ProductsFixtures
+
+    @invalid_attrs %{title: nil}
+
+    test "list_products/0 returns all products" do
+      product = product_fixture()
+      assert Products.list_products() == [product]
+    end
+
+    test "get_product!/1 returns the product with given id" do
+      product = product_fixture()
+      assert Products.get_product!(product.id) == product
```

Comment:
If you test getting product, I’d also test failing to get product with `assert_raise/3` here.